static/js/app.ts

```diff
@@ -230,6 +231,10 @@ export function runit(level: string, lang: string, cb: () => void) {
         }
         return;
       }
+      if (response.Code){
+        console.log("success!");
+        success.show(ErrorMessages['Transpile_success']);
+      }
```

Comment:
I think this should happen after the python program runs. Currently running it may still give an error.